client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -155,7 +155,7 @@ class RegistrantExtraInfoCaForm extends React.PureComponent {
 					<FormLabel>
 						<FormCheckbox
 							id="cira-agreement-accepted"
-							defaultChecked={ true }
+							checked={ ciraAgreementAccepted }
```

Comment:
I wanted it to be checked by default. I don't think `ciraAgreementAccepted` should never be false. I mean, if the user unchecks it we save that, but there's really no point. 
I don't see this as a blocker, just trying to understand what I missed